app/_src/gateway/production/deployment-topologies/hybrid-mode/incremental-config-sync.md

```diff
@@ -0,0 +1,134 @@
+---
+title: Incremental Configuration Sync
+---
+
+In hybrid mode, whenever you make changes to {{site.base_gateway}} entity configuration on the Control Plane, it immediately triggers a cluster-wide update of all Data Plane configurations. 
+In these updates, {{site.base_gateway}} sends the entire configuration set to the Data Planes - therefore, the bigger your configuration set is, the more time it takes to send and process, and the more memory is consumed proportional to the configuration size.
```

Comment:
```suggestion
In these updates, {{site.base_gateway}} sends the entire configuration set to the Data Planes. The bigger your configuration set is, the more time it takes to send and process, and the more memory is consumed proportional to the configuration size. This can result in latency spikes and loss in throughput for high-traffic Data Planes under certain conditions.
```